test/e2e-common.sh

```diff
@@ -28,11 +28,10 @@ INSTALL_ISTIO_CRD_YAML=""
 INSTALL_ISTIO_YAML=""
 # TODO(#2122): Install monitoring as well once we have e2e testing for it.
 INSTALL_RELEASE_YAML=""
+
 # Build is used by some tests and so is also included here.
-# TODO: Should we install build from a nightly release?
-# The latest released Build is always at this location.
-readonly INSTALL_BUILD_YAML="https://storage.googleapis.com/knative-releases/build/latest/release.yaml"
-readonly INSTALL_PIPELINE_YAML="./third_party/config/pipeline/release.yaml"
+readonly INSTALL_BUILD_YAML="./third_party/config/build/"
```

Comment:
nit: not longer points to a yaml

```suggestion
readonly INSTALL_BUILD_DIR="./third_party/config/build/"
```